File: Programming Fundamentals with Python/exam_prep_all/exam_prep_v04/2_destination_mapper.py
import re

user_input = input()

pattern = re.compile("=([A-Z]{1}[A-z]+)=|\/([A-Z]{1}[A-z]+)\/")
locations = []
travel_points = 0

matches = re.findall(pattern, user_input)
for match_ in matches:
    for match in match_:
        if match != '' and len(match) >= 3:
            travel_points += len(match)
            locations.append(match)
            # print(locations)

printable = ', '.join(locations)

print(f"Destinations: {printable}")
print(f"Travel Points: {travel_points}")

File: Programming Fundamentals with Python/exam_prep_all/exam_prep_v04/2_destination_mapper_.js
// ref link for source code: https://softuni.bg/forum/answers/details/61828
// author not me but: Tzigoriyn
function destinationMapper(destination) {
    let pattern = /([=\/])([A-Z]+[a-z]+)\1/gi;
    let count = 0;
    let ourCity = [];

    if (pattern.test(destination)) {
        const ourDest = destination.match(pattern);

        ourDest.forEach(destination => {
            count += (destination.length - 2);
            ourCity.push(destination.slice(1, (destination.length - 1)));

        });
    }

        console.log(`Destinations: ${ourCity.join(", ")}`);
        console.log(`Travel Points: ${count}`);

}